src/geometry/interaction_groups.rs

```diff
@@ -56,24 +77,44 @@ impl InteractionGroups {
         self
     }
 
-    /// Check if interactions should be allowed based on the interaction memberships and filter.
+    /// Check if interactions should be allowed based on the interaction groups and mask.
```

Comment:
this naming change does not correspond to the properties of `InteractionGroup` anymore, if this is intentional, careful rename should appear throughout the crate (and probably a mention in the changelog, then userguides in rapier.rs), I'd be fine with considering that a follow-up.